sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/converters/IndexActionConverter.java

```diff
@@ -56,19 +61,16 @@
             indexAction.setActionType(actionType);
         }
 
-        ObjectMapper mapper = new JacksonAdapter().serializer();
-        SerializationUtil.configureMapper(mapper);
-
         Map<String, Object> additionalProperties;
         TypeReference<Map<String, Object>> typeRef = new TypeReference<Map<String, Object>>() {};
         if (obj.getParamMap() != null) {
             Map<String, Object> properties = obj.getParamMap();
-
-            mapper.setSerializationInclusion(JsonInclude.Include.ALWAYS);
-            additionalProperties = mapper.convertValue(properties, typeRef);
+            MAPPER.setSerializationInclusion(JsonInclude.Include.ALWAYS);
+            additionalProperties = MAPPER.convertValue(properties, typeRef);
         } else {
             T properties = obj.getDocument();
-            additionalProperties = mapper.convertValue(properties, typeRef);
+            MAPPER.setSerializationInclusion(JsonInclude.Include.NON_DEFAULT);
```

Comment:
Possible chance for race conditions here.

Reply:
Good catch. Will have local ObjectMapper here in case other thread set something else.